cmd/crowdsec-cli/metrics.go

```diff
@@ -235,6 +300,10 @@ func ShowPrometheus(url string) {
 		parsersTable.Render()
 		log.Printf("Local Api Metrics:")
 		apilTable.Render()
+		log.Printf("Local Api Machines Metrics:")
+		apilMachinesTable.Render()
+		log.Printf("Local Api Bouncers Metrics:")
+		apilBouncersTable.Render()
 	} else if csConfig.Cscli.Output == "json" {
 		for _, val := range []map[string]map[string]int{acquis_stats, parsers_stats, buckets_stats, apil_stats} {
```

Comment:
I think we want to add the news stats here ?

Reply:
yes